relion/convert/convert_coordinates.py

```diff
@@ -0,0 +1,239 @@
+# **************************************************************************
```

Comment:
Am I right that this is just moved code? Nothing new?

Reply:
Yes, moved code from convert_deprecated mainly...there were a lot of coordinates related functions that might help to have in its own implementation file.